deebot_client/commands/json/clean_logs_v2.py

```diff
@@ -0,0 +1,96 @@
+"""clean log commands."""
+import json
+from typing import Any
+
+from deebot_client.authentication import Authenticator
+from deebot_client.command import CommandResult
+from deebot_client.const import PATH_API_DLN_LOG_LIST, REQUEST_HEADERS
+from deebot_client.event_bus import EventBus
+from deebot_client.events import CleanJobStatus, CleanLogEntry, CleanLogEvent
+from deebot_client.logging_filter import get_logger
+from deebot_client.models import DeviceInfo
+
+from .common import JsonCommand
+
+_LOGGER = get_logger(__name__)
+
+
+class GetCleanLogsV2(JsonCommand):
+    """Get clean logs command."""
+
+    _targets_bot: bool = False
+    name = "clean"
+
+    def __init__(self, count: int = 0) -> None:
+        super().__init__({"count": count})
+
+    async def _execute_api_request(
+        self, authenticator: Authenticator, device_info: DeviceInfo
+    ) -> dict[str, Any]:
+        credentials = await authenticator.authenticate()
+
+        query_params = {
+            "did": device_info.did,
+            "res": device_info.resource,
+            "version": "v2",
+            "logType": self.name,
+            "channel": "google_play",
+            "size": "10",
+            "auth": json.dumps(
+                {
+                    "with": "users",
+                    "userid": credentials.user_id,
+                    "realm": "ecouser.net",
+                    "token": credentials.token,
+                    "resource": device_info.resource,
+                }
+            ),
+            # "country": "DE",
```

Comment:
What are these keys?

Reply:
I note some more params which are set in the " query_params", but i now not have included, if they somehow are needed. 
I current tested this version only with the local bumper server. Possible they are not needed and can be removed.
Also, as mentioned, this is not 100% correct implemented now, as this request is original send over a GET requested and i used the implemented POST request.